.gitignore

```diff
@@ -0,0 +1,2 @@
+ Coverage
```

Comment:
The directory is `coverage`; The casing here might be enough for git to not understand what you want ignored.